src/validation/json-schemas/index.ts

```diff
@@ -2,6 +2,7 @@ import CollectionsQuery from './collections/collections-query.json' assert { typ
 import CollectionsWrite from './collections/collections-write.json' assert { type: 'json' };
 import Definitions from './definitions.json' assert { type: 'json' };
 import GeneralJws from './general-jws.json' assert { type: 'json' };
+import GeneralJsonWebToken2020 from './general-jwk-2020.json' assert {type: 'json'};
```

Comment:
```suggestion
import JwkVerificationMethod from './jwk-verification-method.json' assert {type: 'json'};
```

Reply:
@thehenrytsai should be addressed in latest commit. Thanks!